pkg/stack/loader.go

```diff
@@ -36,50 +34,44 @@ func LoadStacks(dir string, stackService Service) error {
 	if err != nil {
 		return fmt.Errorf("error in stack config: %v", err)
 	}
-	_ = stacks
-
-	entries, err := os.ReadDir(dir)
-	if err != nil {
-		return fmt.Errorf("failed to read stack folder: %s", err)
-	}
-
-	for _, entry := range entries {
-		if !entry.IsDir() {
-			continue
-		}
 
-		name := entry.Name()
-		log.Printf("Parsing stack: %q\n", name)
-		stackTemplate, err := parseStack(dir, name)
-		if err != nil {
-			return fmt.Errorf("error parsing stack %q: %v", name, err)
-		}
+	for name := range stacks {
```

Comment:
Can we not do `name, stack := range stacks` 🤔

Reply:
Please note the todo comment on the next line

Reply:
let go with your solution. In 1.21 we could try to enable https://github.com/golang/go/wiki/LoopvarExperiment